docker-entrypoint.sh

```diff
@@ -19,6 +19,17 @@ if [[ "$APPLY_MIGRATIONS" = "1" ]]; then
 fi
 
 
-# Start server TODO run with uwsgi like tunnistamo?
-python ./manage.py runserver 0.0.0.0:8000
+if [[ "$CREATE_SUPERUSER" = "1" ]]; then
+python ./manage.py shell -c "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.filter(email='admin@example.com').delete(); User.objects.create_superuser('admin', 'admin@example.com', 'adminpass')"
+ echo "Admin user created with credentials admin:adminpass (email: admin@example.com)"
+fi
 
+# Start server
+if [[ ! -z "$@" ]]; then
+    "$@"
+elif [[ "$DEV_SERVER" = "1" ]]; then
+    python ./manage.py runserver 0.0.0.0:8000
+else
+    python ./manage.py collectstatic --noinput
```

Comment:
Now that the images have been moved to the customer UI, we don't need to run this